manifests/instance.pp

```diff
@@ -80,7 +80,7 @@
 #   Default: 64mb
 #
 # [*redis_slave_output_buffer_soft_limit_max_interval*]
-#   Upper bound on the time interval during which the slave replication buffer continuously exceeds the soft limit.  
+#   Upper bound on the time interval during which the slave replication buffer continuously exceeds the soft limit.
```

Comment:
It looks like your editor is clearing trailing whitespace. The PR is clearer if you append ?w=0 on the URL.